### DISCUSSION TITLE
Nominate Jerico Pena to the stacks contributors team

### DISCUSSION THREAD

**Discussion by robdimsdale** [paketo-buildpacks/feedback#205] [Stacks Team]
I'd like to nominate Jerico Pena (@jericop) as a stacks contributor. Jerico has provided valuable perspective on the experience of running on custom stacks, and has contributed a lot to the discussion of multi-arch support. Jerico is willing and able to continue to provide input in the stacks domain.

As Jerico starts to implement arm64 in the stacks, the feedback loop will be much faster for him if is a contributor (no need for first-time PR approvals, etc).

@paketo-buildpacks/stacks-maintainers can I get a 👍 / 👎 ?

**Comment by sophiewigmore** [comment]
@robdimsdale we have 2 votes!

**Reply by robdimsdale** [reply]
@paketo-buildpacks/steering-committee can you add Jerico to the stacks contributors teams please?

**Reply by ForestEckhardt** [reply]
I have invited him. Could you please add him to the [TEAMS.md](https://github.com/paketo-buildpacks/community/blob/main/TEAMS.md)

**Reply by robdimsdale** [reply]
https://github.com/paketo-buildpacks/community/pull/153
